src/components/value-list/value-list.tsx

```diff
@@ -313,31 +327,34 @@ export class ValueList<
     this.filteredItems = filteredItems;
   };
 
-  setUpDragAndDrop(): void {
-    this.cleanUpDragAndDrop();
+  setUpSorting(): void {
```

Comment:
Would this be something we need defined in the interface as well?

Reply:
It could be but it doesn't need to be. I left it out for now since it wouldn't do anything by being added to the interface.